server/routes.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = new express.Router();
+const data = require('../data/data');
+const auth = require('../utils/auth');
+const path = require('path');
+const fs = require('fs');
+
+router.get('/', function (req, res) {
+    res.renderMin('index.ejs', data.getData());
+});
+
+router.get('/modlist/:id', function (req, res) {
+    if (0 <= req.params.id && req.params.id <= data.servers.length) {
+        res.render('importModList.ejs', data.servers[req.params.id]);
+    } else {
+        res.status(404).send('No mod List found');
```

Comment:
SHould you call out `id` which is missing in the error message?

Reply:
no because it bears no resemblance to their request